AAInstrumentation/core.py

```diff
@@ -117,7 +117,7 @@ def generate_baseline(self):
                 "build",
                 str(self.initial_dir),
                 str(self.benchmark),
-                str(self.instr_path.joinpath("clang")),
+                str(self.instr_path / Path("clang")),
```

Comment:
Given that `.instr_path` is already a `Path` you can write this instead: `self.instr_path/"clang"`